ml-agents-envs/mlagents_envs/tests/test_gym_wrapper.py

```diff
@@ -0,0 +1,44 @@
+from mlagents_envs.gym_wrapper import GymToUnityWrapper
+from mlagents_envs.base_env import ActionType
+
+try:
```

Comment:
Similar comment as gym_wrapper.py. Pretty sure that this file will always get imported by pytest, so mlagents-envs will not be runnable on their own.